packages/constants/data/token.json

```diff
@@ -384,6 +384,17 @@
         "Matic": "",
         "Mumbai": ""
     },
+    "maUSDC_ADDRESS": {
```

Comment:
plz remove the limit
https://github.com/DimensionDev/Maskbook/blob/896863d189a894320a45c9837e7dea768ecdcc8b/.vscode/schema/constants.json#L4-L6